distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,79 @@
+priority "grant";
+
+deny {
+  // Read, write to, or execute any file.
+  permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+  permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+
+  // Change current security manager
+  permission java.lang.RuntimePermission "setSecurityManager";
+
+  // Modify application permissions at will.
+  permission java.security.SecurityPermission "setPolicy";
+
+  // Load classes into any protection domain
+  permission java.lang.RuntimePermission "createClassLoader";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/ddf/-" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
+  permission java.io.FilePermission "${ddf.home}", "read, write";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/codice/-" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
+  permission java.io.FilePermission "${ddf.home}", "read, write";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/eclipse/platform/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/apache/karaf/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/apache/aries/blueprint/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+};
+
+grant codeBase "file:/${ddf.home.policy}lib/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant principal org.apache.karaf.jaas.boot.principal.RolePrincipal "manager", principal org.apache.karaf.jaas.boot.principal.RolePrincipal "admin" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
+  permission java.io.FilePermission "${ddf.home}", "read, write";
+
+  permission javax.security.auth.AuthPermission "getSubject";
+};
+
+grant {
```

Comment:
migration export/import needs read access to:
- javax.net.ssl.keyStore
- javax.net.ssl.trustStore

Reply:
Karaf restart functionality requires read and write access to karaf.restart.jvm from both karaf's code and also from our own code where we implement the restart hook for the UI and restart in migration:import.